cmd/crc/cmd/daemon_windows.go

```diff
@@ -17,16 +17,15 @@ func runDaemon() {
 	RunDaemonService("CodeReady Containers", false)
 }
 
-var elog debug.Log
-
 type daemonService struct {
+	eventLogger debug.Log
 }
 
 func (m *daemonService) Execute(args []string, r <-chan svc.ChangeRequest, changes chan<- svc.Status) (ssec bool, errno uint32) {
```

Comment:
preexisting, but `m` is an odd name for a variable of type `*daemonService`

Reply:
done!